labgrid/remote/client.py

```diff
@@ -25,6 +25,7 @@
 from .. import Target, target_factory
 from ..util.proxy import proxymanager
 from ..util.helper import processwrapper
+from ..driver import Mode as WriteMode
```

Comment:
Looking at this we should rename the generic `Mode` to something more generic like `USBStorageWriteMode`.

Reply:
Changing it in usbstoragedriver.py would be a breaking change. Is the change useful enough for that?

Reply:
> Changing it in usbstoragedriver.py would be a breaking change. Is the change useful enough for that?

Hm, I'd like to annotate the old import with a DeprecationWarning, however it's not immediately obvious to me how. If we can't do this, it's not worth it to change it for that.